sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/InstallerUtilsTest.java

```diff
@@ -122,4 +132,61 @@ public void throwIOExceptionWhenTryToCreateSession() throws IOException {
         /* Verify that the session wasn't created. */
         verify(mMockPackageInstaller, never()).openSession(anyInt());
     }
+
+    @Test
+    public void isSystemAlertWindowsEnabledReturnsTrueIfBuildVersionLowerQ() throws Exception {
+        /* Mock SDK_INT to P */
+        setFinalStatic(Build.VERSION.class.getField("SDK_INT"), Build.VERSION_CODES.P);
+
+        /* Set canDrawOverlays to false */
+        mockStatic(Settings.class);
+        when(Settings.canDrawOverlays(any(Context.class))).thenReturn(false);
+
+        /* Check if system alert windows is enabled */
+        assertTrue(InstallerUtils.isSystemAlertWindowsEnabled(mContext));
+
+        /* Set canDrawOverlays to true and make sure it doesn't affect
+        * the return value of isSystemAlertWindowsEnabled.
+        * */
+        when(Settings.canDrawOverlays(any(Context.class))).thenReturn(true);
+
+        /* Check if system alert windows is enabled */
```

Comment:
`Check if system alert windows is enabled` -> `Check if system alert windows is enabled.`